sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AzureMonitorExporterProviderKeys.java

```diff
@@ -0,0 +1,8 @@
+package com.azure.monitor.opentelemetry.exporter.implementation;
```

Comment:
new classes are missing copyrights..  is there a single command to apply it on azure-sdk? try this command:
`mvn clean install -DskipTests -Dgpg.skip -Drevapi.skip -Dmaven.javadoc.skip -Dspotbugs.skip -f sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml`

Reply:
(I used `grep -r --include=*.java -L Copyright`)